### ISSUE TITLE
Hard crash when closing "Planets Tonight" modal before it loads 

### ISSUE THREAD

**Issue by BradleyPelton** [Stellarium/stellarium-web-engine#49]
### Summary
TLDR: https://gyazo.com/f194c1db93728d144309e56c1ab303b1
SUMMARY: When a user clicks the "Planets Tonight" button and quickly clicks away(before it finishes loading), the entire app hard crashes(crashes and never recovers, completely unresponsive).
PRIORITY: High Priority

NOTES: Interestingly, there are no  errors thrown in the console.
NOTES: 100% reproducible on my machine. Please feel free to ping me if you can't reproduce it.

### Expected Behavior
The "Planets Tonight" modal either opens and then closes, or never opens in the first place.

### Actual Behavior
The web app crashes when a user tries to close the "Planets Tonight" modal before it loads.

### Steps to reproduce:
(NOTE: 100% reproducible, read carefully if you can't reproduce)
1.) Refresh
2.) Double click the "Planets Tonight" menu button in the top left
3.) Before the "Planets Tonight" modal finishes loading, click the center of the page
4.) App will hard crash

### Gif of execution
https://gyazo.com/f194c1db93728d144309e56c1ab303b1

**Comment by xalioth** [comment]
Hi,

I can't reproduce this one in last release.
